external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -138,6 +138,7 @@ pub struct ModuleDefinition {
 #[derive(Debug, Clone)]
 pub struct Friend {
     pub attributes: Attributes,
+    pub attr_locs: Vec<Loc>,
```

Comment:
Is this not already present in the `Attributes` map?

Reply:
No,  the overall spans are gone after `flatten_attributes` happens; only the actual attribute text spans are left. This seemed easier than reworking `flatten_attribute` and all of its callers.